index.js

```diff
@@ -92,7 +92,8 @@ function Peer (opts) {
   try {
     self._pc = new (self._wrtc.RTCPeerConnection)(self.config)
   } catch (err) {
-    self.destroy(err)
+    process.nextTick(() => self.destroy(err))
```

Comment:
Is this browserified nicely for browsers? Maybe use `setImmediate()` or `setTimeout()` instead?

Reply:
It is handled nicely by `browserify` but it adds 2245 bytes to the bundle size. Unfortunately, `setImmediate` is non-standard.

Reply:
We use setTimeout for this purpose elsewhere already.